src/main/java/org/assertj/core/internal/Numbers.java

```diff
@@ -51,8 +49,8 @@ public Numbers(ComparisonStrategy comparisonStrategy) {
   /**
    * Asserts that the actual value is equal to zero.<br>
    * It does not rely on the custom comparisonStrategy (if one is set).
-   * 
-   * @param info contains information about the assertion.
+   *
+   * @param info   contains information about the assertion.
```

Comment:
please revert al the unwanted formatting, this makes this PR hard to review because 90% of the changes in this file are irrelevant.
https://github.com/assertj/assertj-core/blob/main/CONTRIBUTING.md indicates how to get the proper formatting rules.

Reply:
I have fixed the formatting problem. Thx!